s4e-web/cypress/plugins/cy-ts-preprocessor.js

```diff
@@ -11,7 +11,10 @@ const webpackOptions = {
         exclude: [/node_modules/],
         use: [
           {
-            loader: 'ts-loader'
+            loader: 'ts-loader',
+            options: {
+              transpileOnly: true
```

Comment:
Why memory leak occurred? To talk.

Reply:
Think about cons and pros of the `transpileOnly` flag
https://github.com/TypeStrong/ts-loader#loader-options